### PR TITLE
Removes some strings that should be typepaths, and some partial pathing, regarding bullets

### PR DESCRIPTION
**Pull request by MadmanMartian**
~~The rounds that were removed weren't used in anything but bullet-pickers (xenoarch guns, roulette revolvers)~~

The partial pathing is related to the booze bullets

### CODE REVIEW

**Comment by SonixApache** [comment]
???

~~Isn't shrapnel used for IEDs and grenades?~~
Shrapnel is used in surgically removing bullets from people, to find out what kind they are. Among other things.

**Comment by Irradiation** [comment]
.666 and .418 (Cyanide and CO2 rounds) have been in the Goonsat loot room for years now.

**Comment by MadmanMartian** [comment]
Aw for.

Compiler never caught those because of string2path

**Comment by SonixApache** [comment]
~~Burst bullets are used by the gyrojet, which is a summon guns thing and ammo for it spawns in maint sometimes.~~
Wait, no they're not. That's weird, I could've sworn they were.

**Comment by Irradiation** [comment]
Shrapnel is what glass shards turn into when you shove them in an IED, yeah.

**Comment by MadmanMartian** [comment]
Something, something

usawnuthin

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -174,21 +186,12 @@
...
-	name = "stunshot"
-	icon_state = "sshell"
-	damage = 5
-	stun = 10
-	weaken = 10
-	stutter = 10
-
-/obj/item/projectile/bullet/a762
-	damage = 25
+/obj/item/projectile/bullet/burstbullet/on_hit(var/atom/target, var/blocked = 0)
```
Comment (MadmanMartian) [inline code/modules/projectiles/projectile/bullets.dm:192]: Adds the on_hit for the exploding bullet. Not as good as the gyrojet rounds, but it's something.

**Comment by Kurfursten** [comment]
looks fine, all set?

**Comment by MadmanMartian** [comment]
should be ready

**Pull request merged**